src/config.js

```diff
@@ -1,8 +1,12 @@
 export default {
-  GOAL_BUFFER_LENGTH: 60,
+  GOAL_BUFFER_LENGTH: 30,
+  MAX_GOAL_BUFFER_LENGTH: 60,
+  GOAL_BUFFER_RATE: 1,
   // A fudge factor to apply to advertised playlist bitrates to account for
   // temporary flucations in client bandwidth
   BANDWIDTH_VARIANCE: 1.2,
   // How much of the buffer must be filled before we consider upswitching
-  BUFFER_LOW_WATER_LINE: 30
+  BUFFER_LOW_WATER_LINE: 0,
+  MAX_BUFFER_LOW_WATER_LINE: 30,
+  BUFFER_LOW_WATER_RATE: 1
```

Comment:
I like separating low waterline and goal buffer length variables for finer control.

As for time vs rate, I think it would be worth testing both.